src/components/IndexTable/IndexTable.scss

```diff
@@ -224,6 +241,39 @@ $loading-panel-height: rem(53px);
   }
 }
 
+.Table-unselectable {
+  .TableCell:first-child {
+    left: 0;
+    background-color: var(--p-surface);
+    position: sticky;
+    z-index: z-index(sticky-cell, $index-table-stacking-order);
+  }
+
+  .statusSuccess {
+    .TableCell:first-child {
+      background-color: var(--p-surface-primary-selected);
+    }
+  }
+
+  .statusSubdued {
+    .TableCell:first-child {
+      background-color: var(--p-surface-subdued);
+    }
+  }
+
+  .TableRow-hovered {
+    .TableCell:first-child {
+      background-color: var(--p-surface-selected-hovered);
```

Comment:
This introduces an issue where the first column is highlighted in blue and the rest are gray on hover.

<img width="1025" alt="Screen Shot 2021-08-23 at 9 46 31 AM" src="https://user-images.githubusercontent.com/3619012/130458816-bb8e0f7b-40c3-405f-b209-22e2519cddd6.png">

Reply:
There was a recent change on these bg colors so it should be whichever one was used here https://github.com/Shopify/polaris-react/pull/4359